lib/govuk_publishing_components/presenters/select.rb

```diff
@@ -0,0 +1,42 @@
+module GovukPublishingComponents
+  module Presenters
+    class SelectHelper
+      attr_reader :options, :option_markup, :selected_option
+
+      def initialize(local_assigns)
+        @options = local_assigns[:options] if local_assigns[:options]
+        @option_markup = get_options
+      end
+
+    private
+
+      def get_options
+        return {} if @options.nil?
```

Comment:
We probably want to return an array here if that's what `formatted_options` is returned as.

We don't need to put `@` here actually as the `attr_reader` is used above.